x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -408,7 +401,7 @@ export const SEARCH_RULES = i18n.translate(
 export const SEARCH_PLACEHOLDER = i18n.translate(
   'xpack.securitySolution.detectionEngine.rules.allRules.searchPlaceholder',
   {
-    defaultMessage: 'e.g. rule name',
+    defaultMessage: 'Search by rule name, index pattern, or MITRE ATT&CK tactic or technique',
```

Comment:
May want the `™` as we're adding it elsewhere in the UI:


<p align="center">
  <img width="700" src="https://user-images.githubusercontent.com/2946766/160500275-9fa0079c-7d4d-4028-80a7-45cd07339039.png" />
</p>

```suggestion
    defaultMessage: 'Search by rule name, index pattern, or MITRE ATT&CK™ tactic or technique',
```